submission/submission-core/src/main/kotlin/ac/uk/ebi/biostd/submission/submitter/request/SubmissionRequestProcessor.kt

```diff
@@ -43,12 +45,16 @@ class SubmissionRequestProcessor(
             logger.info { "$accNo ${sub.owner} Finished persisting file ${rqtFile.index}, path='${rqtFile.path}'" }
         }
 
+        val mutexMap = ConcurrentHashMap<String, Mutex>()
```

Comment:
I do not think this is necesary any more. I mean we will only re use file if it is in the same path so there is not race condition posibility.